opencv-stitching-jlink/README.md

```diff
@@ -1,11 +1,11 @@
 Sample project for using OpenCV from a generated Java binding made with [Bytedeco's](http://bytedeco.org/) JavaCPP technology.
- * A jlink version of the [original stitching sample project](../opencv-stitching), using the maven-jlink plugin.
+ * A jlink version of the [original stitching sample project](../opencv-stitching), using the Apache Maven JLink plugin.
 
 On the command line, run:
 
 ```bash
-$ mvn clean package -Djavacpp.platform.custom -Djavacpp.platform.host
-$ ./target/image/bin/stitch panorama_image1.jpg panorama_image2.jpg --output panorama_stitched.jpg
+$ mvn clean package -Djavacpp.platform.host
```

Comment:
We should leave `-Djavacpp.platform.custom` here. `-Djavacpp.platform.host` won't always work correctly without it.

Reply:
Re-added.
But for my information, what can turn wrong without it ?

Reply:
The profiles for the `javacpp.platform` one don't get disabled without it:
https://github.com/bytedeco/javacpp-presets/wiki/Reducing-the-Number-of-Dependencies